src/components/hero/campaignHero.tsx

```diff
@@ -1,24 +1,52 @@
 import { Hero, Link as DSLink } from "@nypl/design-system-react-components";
-import { imageURL } from "../../utils/utils";
 import CampaignHeroSubText from "./campaignHeroSubText";
 import CampaignHeroHeading from "./campaignHeroHeading";
+import CampaignHeroLoading from "./campaignHeroLoading";
+import { useEffect, useState } from "react";
 
-const CampaignHero = ({ featuredItem, numberOfDigitizedItems }: any) => {
-  return (
+const CampaignHero = ({ imageID }) => {
+  console.log(imageID);
+  // const [isFeaturedItemLoading, setIsFeaturedItemLoading] = useState(false);
+  const [data, setData] = useState<any>({});
```

Comment:
As discussed, this is pretty much the same as the `isFeaturedItemLoading` variable since this shows that the data is retrieve and available. Okay to just use `data` and check of the updated data's property(ies).